neuralmonkey/nn/projection.py

```diff
@@ -64,3 +67,17 @@ def maxout(inputs, size, scope="MaxoutProjection"):
 
         reshaped = tf.reshape(maxpooled, [-1, size])
         return reshaped
+
+
+def multilayer_projection(input_, layer_sizes, activation=tf.nn.relu,
```

Comment:
tahle funkce dela neco jineho, nema vystupni vrstvu. Resil jsem to vcera s @jlibovicky